python_modules/libraries/dagster-fivetran/dagster_fivetran/resources.py

```diff
@@ -592,6 +594,57 @@ def get_groups(self) -> Mapping[str, Any]:
         """
         return self._make_request("GET", "groups")
 
+    def update_schedule_type_for_connector(
+        self, connector_id: str, schedule_type: str
+    ) -> Mapping[str, Any]:
+        """Updates the schedule type property of the connector to either "auto" or "manual".
+
+        Args:
+            connector_id (str): The Fivetran Connector ID. You can retrieve this value from the
+                "Setup" tab of a given connector in the Fivetran UI.
+            schedule_type (str): Either "auto" (to turn the schedule on) or "manual" (to
+                turn it off).
+
+        Returns:
+            Dict[str, Any]: Parsed json data representing the API response.
+        """
+        schedule_types = {s for s in FivetranConnectorScheduleType}
+        if schedule_type not in schedule_types:
+            check.failed(
```

Comment:
nit; include a reference to the connector in question

Reply:
Done in [c236554](https://github.com/dagster-io/dagster/pull/25911/commits/c23655472fef69196bdfa933730a3a578a9640e9)